dicto/rules

```diff
@@ -141,4 +147,33 @@ WholeIliasCodebaseExceptInitialisation cannot depend on: GlobalsExceptDIC
  * mostly points to some underlying problem needing to be solved instead of using this.
  * This was introduced as temporary workaround. See: https://github.com/ILIAS-eLearning/ILIAS/pull/1064
  */
-WholeIliasCodebase cannot depend on: IsDependencyAvailableMethod
\ No newline at end of file
+WholeIliasCodebase cannot depend on: IsDependencyAvailableMethod
+
+/**
+ * ilTemplate::getMessageHTML is a method that allows consumers to render a
+ * message-box according to the standard format. The kitchen sink component
+ * Message Box was introduced with ILIAS 5.4 and should be used instead.
+ */
+WholeIliasCodebase cannot depend on: GetMessageHTML
+
+/**
+ * ilGlobalTemplate::setOnScreenMessage is only supposed to be used by ilUtil.
+ */
+WholeIliasCodebaseExceptIlUtil cannot depend on: SetOnScreenMessage
+
+/**
+ * The method creates HTML to display a system message. It is to be replaced by
```

Comment:
This would be possible if the UX/Layout/Whatever service offers a named slot a `MessageBox` could be passed to.

Reply:
I don't think you should/can change something **here**, but this should be somehow added to a kind of roadmap (okay, this is implicitly done by the new *DICTO* rules)